SudokuBLL/src/test/java/pkgGame/SudokuPrivateMethodsTest.java

```diff
@@ -16,6 +16,95 @@ private void PrintStars() {
 			System.out.print("*");
 		System.out.println();
 	}
+	
+	@Test
+	public void isDifficultyMet_Test1() throws NoSuchMethodException, InvocationTargetException {
+		
+		// expecting true, difficulty should easily be met
+		
+		Sudoku s1 = null;
+		int iPuzzleSize = 9;
+		boolean iExpectedMet = true;
+		
+		boolean iActualMet;
+
+		try {
+			Class<?> c = Class.forName("pkgGame.Sudoku");
+			Constructor constructor = c.getConstructor(new Class[] { int.class, eGameDifficulty.class });
+			constructor.setAccessible(true);
+			s1 = (Sudoku) constructor.newInstance(iPuzzleSize, eGameDifficulty.EASY);
+
+			Method misDifficultyMet = c.getDeclaredMethod("isDifficultyMet", new Class[] { int.class });
+
+			misDifficultyMet.setAccessible(true);
+			iActualMet = (boolean)misDifficultyMet.invoke(s1, 9000);
+
+			assertTrue(iExpectedMet == iActualMet);
```

Comment:
You could just say `assertTrue(iActualMet)`, but, I guess you could delineate it like this.